src/components/Form.js

```diff
@@ -1,19 +1,15 @@
 import React, { useState } from 'react'
 import { connect } from 'react-redux'
 import { addTodo } from '../store/todos'
-
-const seed = 999999
-
-const generateId = () => {
-  return Math.floor(Math.random() * seed)
-}
+import { nanoid } from 'nanoid'
 
 const Form = (props) => {
   const [value, setValue] = useState(' ')
+  const { addTodo } = props
   async function handleSubmit (event) {
     event.preventDefault()
-    const newTodo = { id: generateId(), item: value, done: false }
-    await props.addTodo(newTodo)
+    const newTodo = { id: nanoid(), item: value, done: false }
```

Comment:
This is good here for now, but eventually you may want to move id generation to the server. The server, on `POST /todos` can return the generated id, which you can either backfill into the object client side, or use it to `GET /todos/:id`, to then add that result to your redux store.

Reply:
I'll create an issue for this so the work isn't lost

Reply:
done in #16